### ISSUE TITLE
A11y_VS Code: Copilot_Suggestion_Keyboard: Tooltip is not accessible using keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#188260]
#A11yTCS; #A11ySev2; #DesktopApp; #Win11; #Win32; #A11yMAS; #SH-VS Code Copilot-Win32-July23; #Keyboard; #A11yWCAG2.1; #Visual Studio Code Client;

### Environment Details:
Application Name: VS Code: Copilot
Visual studio code Insiders Version: 1.81.0(User Setup)
Microsoft Windows 11 Enterprise 22H2 Build 22621.1992

### Repro Steps:

1. Open the "Visual Studio Code" dialog box.
2. In Visual Studio Code, create a new JavaScript (*.js) file.
3. In the code Paste this code function calculateDaysBetweenDates(begin, end) { and press enter.
4. A suggestion will appear.
5. Prees "Ctrl + K, Ctrl + I" and a tooltip will appear.
6. Verify whether the tooltip is accessible using keyboard or not.

### Actual Result:​
Tooltip is not accessible using keyboard.

### Expected Result:​
Tooltip should be accessible using keyboard.

### User Impact:
Keyboard users will face difficulty in accessing the tooltip info.

### Attachment:

![Bug_188260_Keyboard](https://github.com/microsoft/vscode/assets/124244478/20cd4bad-3d52-4704-bb8d-54170ccbebff)

**Comment by hediet** [comment]
There is a shortcut to move focus to the hover!

**Comment by AnKushSingh05** [comment]
@hediet Could you please provide us the shortcut key ,so that we can verify and close the bug.

**Comment by isidorn** [comment]
The shortcut is `ctrl+k, ctrl+i`. If you are in screen reader mode the focus will be passed automatically. To turn on accessibility mode just have a screen reader attached, or put `"editor.accessibilitySupport": "on"` in vscode settings.

**Comment by AnKushSingh05** [comment]
@isidorn With ctrl+k, ctrl+i we can enable the tooltip but with the keyboard, focus is not going to the content present on the tooltip. Is there any method by which we can all the content on the tooltip like this accept line?.
![image](https://github.com/microsoft/vscode/assets/124244478/9f8fe8cf-3e97-49a2-be3f-fb2cf3f831b4)

**Comment by isidorn** [comment]
@hediet I also can not move the focus to the ghost text hover with Ctrl+K, Ctrl+I. Might this be a regression? Let me reopen so you double check. Thank you 🙏

**Comment by meganrogge** [comment]
@hediet lmk if you want help.

**Comment by AnKushSingh05** [comment]
Regress the issue in version: 1.82.0-insider (user setup).
Issue is fixed. Hence closing the bug.
PFA for the reference.

![Bug_188260_Fixed](https://github.com/microsoft/vscode/assets/124244478/6e1f82f0-bddc-4c95-b5fe-e2446645c59c)
